### PR TITLE
Improved translations for Kannada

### PR DESCRIPTION
**Pull request by developergouli**
**Description of PR**
Fixed the translations for Kannada language

**Type of PR**

- [ ] Bugfix
- [x] New feature

**Relevant Issues**  
Fixes #...
#432 
#357 
**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**

**Desktop**

image

image

**Mobile (iPhone)**

image

image

### CODE REVIEW

File: public/locales/kn.json [hunk 1]
```diff
@@ -305,44 +305,44 @@
...
+      "Ramgarh": "ರಾಮಗಢ",
       "Ranchi": "ರಾಂಚಿ",
       "Sahibganj": "ಸಾಹಿಬ್‌ಗಂಜ್",
       "Saraikela Kharsawan": "ಸರೈಕೆಲಾ ಖರ್ಸವಾನ್",
       "Simdega": "ಸಿಮ್ಡೆಗಾ"
     },
     "Karnataka": {
       "title": "ಕರ್ನಾಟಕ",
-      "Chikmagalur": "ಚಿಕ್ಮಗಲೂರ್",
+      "Chikmagalur": "ಚಿಕ್ಮಮಗಳೂರು",
```
Comment (sumanthbhatt) [inline public/locales/kn.json]: ಚಿಕ್ಕಮಗಳೂರು ?

File: public/locales/kn.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+  "Checked by default to scale uniformly": "ಸಮ ಅಂತರದಲ್ಲಿ ಬಿಂಬಿಸಲು ಪೂರ್ವನಿಯೋಜಿತವಾಗಿ ಆಯ್ಕೆ ಮಾಡಿದೆ",
   "Last Updated": "ಕೊನೆಯದಾಗಿ ನವೀಕರಿಸಲಾಗಿದೆ",
-  "Daily": "ದೈನಂದಿನ",
-  "Confirmed": "ದೃ .ಪಡಿಸಲಾಗಿದೆ",
+  "Daily": "ದೈನಿಕ",
+  "Confirmed": "ಖಚಿತ",
   "Active": "ಸಕ್ರಿಯ",
-  "Recovered": "ಚೇತರಿಸಿಕೊಂಡ",
-  "Deceased": "ಮರಣ",
+  "Recovered": "ವಾಸಿ",
```
Comment (sumanthbhatt) [inline public/locales/kn.json]: Prefer ಗುಣವಾದವರು here. Vaasi is mostly south Karanataka specific word.

File: public/locales/kn.json [hunk 3]
```diff
@@ -1,21 +1,21 @@
 {
-  "We stand with everyone fighting on the frontlines": "ಮುಂಚೂಣಿಯಲ್ಲಿ ಹೋರಾಡುವ ಪ್ರತಿಯೊಬ್ಬರೊಂದಿಗೆ ನಾವು ನಿಲ್ಲುತ್ತೇವೆ",
-  "Cumulative": "ಸಂಚಿತ",
-  "Spread Trends": "ಟ್ರೆಂಡ್‌ಗಳನ್ನು ಹರಡಿ",
+  "We stand with everyone fighting on the frontlines": "ಮುಂದೆ ನಿಂತು ಹೋರಾಡುತ್ತಿರುವ ಪ್ರತಿಯೊಬ್ಬರೊಂದಿಗೆ ನಾವಿದ್ದೇವೆ",
+  "Cumulative": "ಒಟ್ಟು",
+  "Spread Trends": "ಹರಡುವಿಕೆ ದಿಕ್ಕು",
```
Comment (sumanthbhatt) [inline public/locales/kn.json:4]: +1

File: public/locales/kn.json (hunk 1 above)
Comment (developergouli) [reply public/locales/kn.json]: Thanks @sumanthbhatt. You are correct. Will correct it in my next PR

File: public/locales/kn.json (hunk 2 above)
Comment (developergouli) [reply public/locales/kn.json]: Had used ವಾಸಿ since it was short. Yes, will use ಗುಣವಾದವರು in my next PR

File: public/locales/kn.json (hunk 1 above)
Comment (sumanthbhatt) [reply public/locales/kn.json]: Super; I'll get my setup ready for some Kannada translations as well.

We need the "About/bagge" page also to be translated. I don't see an option in the kn.json file to do it. Is this an item in the list or did I miss something ?

File: public/locales/kn.json (hunk 2 above)
Comment (sumanthbhatt) [reply public/locales/kn.json]: Checked with couple of Kannada friends; they suggested guNamukha - this should be shorter I hope.

File: public/locales/kn.json (hunk 1 above)
Comment (developergouli) [reply public/locales/kn.json]: The data for About page directly flows from API. The source code for API is available here
https://github.com/covid19india/api

File: public/locales/kn.json (hunk 2 above)
Comment (developergouli) [reply public/locales/kn.json]: Sure. Point noted

File: public/locales/kn.json (hunk 1 above)
Comment (developergouli) [reply public/locales/kn.json]: Have corrected the spelling for ಚಿಕ್ಕಮಗಳೂರು

File: public/locales/kn.json (hunk 2 above)
Comment (developergouli) [reply public/locales/kn.json]: Changed to ಗುಣಮುಖ now

**Comment by bee-rickey** [comment]
Can we keep it as "Tracker" in Kannada than "Jaadugaara"?

**Comment by sumanthbhatt** [comment]
@Jeremyphilemon can you please merge the latest changes in this PR? (+1 from my side to all changes here)

**Comment by memetics19** [comment]
it's better to translate the hours and minutes to kannada as "nimishagalu" and "ghantegalu"

**Comment by developergouli** [comment]
> it's better to translate the hours and minutes to kannada as "nimishagalu" and "ghantegalu"

Thanks Shreeda for the comments. 

Looks like the duration details (hours and minutes) are coming from one of the javascript library (date-fns). As of today, this library does not have support for Kannada language. Hence this change might not be possible in current design

**Comment by developergouli** [comment]
> Jaadugaara

Thanks bee-rickey for the comments. I would be happy to make the change. However my initial thought process here was to use as much Kannada words as possible. 

To be aligned with this thought process, any suggestion about simple Kannada word for tracker from your side would be highly appreciated.

**Comment by memetics19** [comment]
i think even the  duration details (date and time) can be translated to kannada by using toLocalstring method

**Review by satvikmaker — APPROVED** [review]

**Pull request closed without merging**